release-maven-central.sh

```diff
@@ -65,5 +65,6 @@ maven_publish "http4k-server-jetty"
 maven_publish "http4k-server-netty"
 maven_publish "http4k-server-undertow"
 maven_publish "http4k-serverless-lambda"
+maven_publish "http4k-testing-chaos"
```

Comment:
For the moment we need to remove this line from the commit - because there is a cycle for when we create a new module - it needs to be uploaded to bintray, then we have to ask permission for it to be linked to JCenter, and only then can we sync it to maven. If we add this line now, it'll fail build that we release it in! :)